src/Abstractions/NexusMods.Abstractions.Loadouts.Synchronizers/ALoadoutSynchronizer.cs

```diff
@@ -750,48 +838,110 @@ public virtual async Task BackupNewFiles(GameInstallation installation, FileTree
         // TODO: This may be slow for very large games when other games/mods already exist.
         // Backup the files that are new or changed
         var archivedFiles = new ConcurrentBag<ArchivedFileEntry>();
-        await Parallel.ForEachAsync(fileTree.GetAllDescendentFiles(), async (file, cancellationToken) =>
+        await Parallel.ForEachAsync(files, async (file, cancellationToken) =>
         {
-            if (file.Item.Value is StoredFile storedFile)
-            {
-                var path = installation.LocationsRegister.GetResolvedPath(storedFile.To);
-                if (await _fileStore.HaveFile(storedFile.Hash))
-                    return;
+            var path = installation.LocationsRegister.GetResolvedPath(file.To);
+            if (await _fileStore.HaveFile(file.Hash))
+                return;
 
-                var archivedFile = new ArchivedFileEntry
-                {
-                    Size = storedFile.Size,
-                    Hash = storedFile.Hash,
-                    StreamFactory = new NativeFileStreamFactory(path),
-                };
+            var archivedFile = new ArchivedFileEntry
+            {
+                Size = file.Size,
+                Hash = file.Hash,
+                StreamFactory = new NativeFileStreamFactory(path),
+            };
 
-                archivedFiles.Add(archivedFile);
-            }
+            archivedFiles.Add(archivedFile);
         });
 
         await _fileStore.BackupFiles(archivedFiles);
     }
+    
+    /// <summary>
+    /// True if the file is marked as deleted.
+    /// </summary>
+    public bool IsDeletedFile(File.Model file)
+    {
+        return file.TryGet(DeletedFile.Deleted, out var deleted) && deleted;
+    }
+
+    /// <summary>
+    /// True if the file is a stored file.
+    /// </summary>
+    public bool IsStoredFile(File.Model file)
+    {
+        return file.Contains(StoredFile.Hash);
+    }
```

Comment:
As written multiple times already, this is inconsistent with everything else and somewhat duplicate with existing methods.